onnx_tf/handlers/backend/cast.py

```diff
@@ -3,31 +3,18 @@
 from onnx_tf.handlers.backend_handler import BackendHandler
 from onnx_tf.handlers.handler import onnx_op
 from onnx_tf.handlers.handler import tf_func
-from onnx_tf.handlers.handler import partial_support
-from onnx_tf.handlers.handler import ps_description
-
 
 @onnx_op("Cast")
 @tf_func(tf.cast)
-@partial_support(True)
-@ps_description("Cast string to float32/float64/int32/int64 " +
-                "are not supported in Tensorflow.")
```

Comment:
Sorry for the delayed review. I believe we still have limited support because cast from a string to another data type depends on tf.strings.to_number, which supports four output types, tf.float32, tf.float64, tf.int32, tf.int64. The currently description states the opposite. Maybe change the wording to something like "Cast string to data types other than float32/float64/int32/int64 is not supported in Tensorflow"?

Reply:
You're right @chinhuang007. I should have caught that.  We are dependent on tf.string.to_number for now.  I have made the update. Thanks